# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../siefs/dont_test_fuse_version.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./configure.in.orig	2005-02-09 11:24:21.713397650 -0300
+++ ./configure.in	2005-02-09 11:25:31.069420341 -0300
@@ -39,14 +39,6 @@
 ])
 fi
 
-if ! grep -q '\(\*release\)' $fuseinst/include/fuse.h ; then
-	AC_MSG_RESULT([old])
-	AC_MSG_ERROR([
-*** You need fuse version 1.0 or later.
-*** Please go to http://sourceforge.net/projects/avf
-*** and download the latest version
-])
-fi
 AC_MSG_RESULT(${fuseinst})
 AC_SUBST(fuseinst)
 
--- ./siefs/Makefile.am.orig	2005-02-09 11:31:40.016076350 -0300
+++ ./siefs/Makefile.am	2005-02-09 11:33:11.394638747 -0300
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-CFLAGS = -I$(fuseinst)/include
+CFLAGS = -I$(fuseinst)/include -D_FILE_OFFSET_BITS=64 -DFUSE_USE_VERSION=11
 
 bin_PROGRAMS = siefs slink
 
